tools/bumper/cnao_repo_commands.go

```diff
@@ -67,6 +68,35 @@ func (cnaoRepoOps *gitCnaoRepo) updateComponentsConfig(relativeConfigPath string
 	return updateComponentsYaml(configPath, componentsConfig)
 }
 
+// resetRepo is a wrapper for resetRepoInAllowedList
+func (cnaoRepoOps *gitCnaoRepo) resetRepo() error {
```

Comment:
Let's rename to `reset` since Repo is part of the struct so we have `cnaoRepo.reset()`

Reply:
Done